src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/MethodTable.Runtime.cs

```diff
@@ -11,25 +11,6 @@ namespace Internal.Runtime
     // Extensions to MethodTable that are specific to the use in Runtime.Base.
     internal unsafe partial struct MethodTable
     {
-#if !INPLACE_RUNTIME
-        internal MethodTable* GetArrayEEType()
-        {
-            MethodTable* pThis = (MethodTable*)Unsafe.Pointer(ref this);
-            void* pGetArrayEEType = InternalCalls.RhpGetClasslibFunctionFromEEType(pThis, ClassLibFunctionId.GetSystemArrayEEType);
```

Comment:
`GetSystemArrayEEType` should be deleted from everywhere if this is getting deleted.